swedish-oidc-profile.md

```diff
@@ -215,7 +215,21 @@ RP gives the OpenID Provider a more exact view of the user being authenticated.
 
 An OpenID Provider compliant with this profile MUST support Request Object JWT:s sent by value (using the `request` parameter) and MAY support Request Object JWT:s sent by reference (using the `request_uri` parameter).
 
-Request objects MUST be signed by the Relying Party's registered key, and they MAY be encrypted to the OpenID Provider's public key.
+An OpenID Provider MUST be prepared to accept and process signed and/or encrypted Request Objects.
+
+A Relying Party that signs a Request Object MUST do so using is registered key, and a Relying Party
```

Comment:
OpenID Connect Core 1.0 sektion 10.1 specifies that:

`If there are multiple keys in the referenced JWK Set document, a kid value MUST be provided in the JOSE Header. `

Personally, I would like all signed request objects, ID tokens, signed userinfo etc. to ALWAYS have a "kid" value present in the JOSE header. JWK sets may change over time.

Could we at least RECOMMEND in this spec to always include "kid" in JOSE headers for signed JWTs?

Reply:
@eriklupander That kind of writing would not fit in this section. Could you make an issue of your comment. Perhaps better that we include such a writing under the "security requirements" section?

Reply:
@martin-lindstrom  Absolutely!